fusedrug_examples/interaction/drug_target/affinity_prediction/PLM_DTI/configs/config.yaml

```diff
@@ -34,7 +34,7 @@ model:
   target_featurizer: ProtBertFeaturizer
   # possible values: SimpleCoembedding, others (not tested): GoldmanCPI, SimpleCosine, AffinityCoembedInner, CosineBatchNorm, LSTMCosine, DeepCosine,
   #                  SimpleConcat, SeparateConcat, AffinityEmbedConcat, AffinityConcatLinear
-  model_architecture: SimpleCoembedding
+  model_architecture: deepcoembedding
```

Comment:
is this intentional in this PR ?

Reply:
I'll revert it, we ended up disabling this example cause it failed.